OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -75,12 +75,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </decoNote>
                                 <decoNote xml:id="b3" type="Cover" color="brown">
                                     The leather cover was decorated with blind tooled ornament (double lines).
-                                    <!-- Description of the tooled ornamentation will be revise --></decoNote>
+                                    <!-- Description of the tooled ornamentation will be revised --></decoNote>
                                 <decoNote xml:id="b4" type="SewingStations">4</decoNote>
                                 <decoNote xml:id="b5" type="Spine">The spine cover is missing.</decoNote>
                                 <decoNote xml:id="b6" type="Other">
                                     Threads of various origin are inserted in the outer margins for navigating the text. 
```

Comment:
Hmm, should we say material rather than origin? @ElianaDalSasso

Reply:
Perhaps in this context "threads of various colours" works